bin/libs/fs.sh

```diff
@@ -32,19 +32,9 @@ $(echo "${children}")
 EOF
 }
 
-# Takes in the file that should be expanded and echos out the result, which the caller needs to read
 convert_to_absolute_path() {
-  file="${1}"
-
-  # If the value starts with a ~ for the home variable then evaluate it
-  file=$(echo "${file}")
-
-  # If the path is relative, then expand it
-  if [[ "${file}" != /* ]]; then
-    file="$(pwd)/${file}"
-  fi
-
-  echo "${file}"
+  orgPath="${1}"
+  echo "$(cd "$(dirname "$orgPath")"; pwd)/$(basename "$orgPath")"
```

Comment:
Few tests you can see the inconsistency:

```
ZOWEAD3:/u/zowead3: >orgPath=/var
ZOWEAD3:/u/zowead3: >echo "$(cd "$(dirname "$orgPath")"; pwd)/$(basename "$orgPath")"
//var
ZOWEAD3:/u/zowead3: >orgPath=/var/zosmf
ZOWEAD3:/u/zowead3: >echo "$(cd "$(dirname "$orgPath")"; pwd)/$(basename "$orgPath")"
/S0W1/var/zosmf
ZOWEAD3:/u/zowead3: >orgPath=../../ZOWE
ZOWEAD3:/u/zowead3: >echo "$(cd "$(dirname "$orgPath")"; pwd)/$(basename "$orgPath")"
//ZOWE
ZOWEAD3:/u/zowead3: >orgPath=.
ZOWEAD3:/u/zowead3: >echo "$(cd "$(dirname "$orgPath")"; pwd)/$(basename "$orgPath")"
/u/zowead3/.
```

Reply:
fixed